File: LeetCode-Programs/Find Triangular Sum of an Array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        while len(nums)>1:
            newNums=[]
            for i in range(len(nums)-1):
                temp=(nums[i]+nums[i+1])%10
                newNums.append(temp)
            nums=newNums
        return nums[0]

File: LeetCode-Programs/Find Triangular Sum of an Array.cpp
class Solution {
public:
    int triangularSum(vector<int>& nums) {
        int temp;
        while(nums.size()>1){
            vector<int> newNums;
            for(int i=0;i<nums.size()-1;i++){
                temp=(nums[i]+nums[i+1])%10;
                newNums.push_back(temp);
            }
            nums=newNums;
        }
        return nums[0];
    }
};
